### ISSUE TITLE
Minor breakage from theme PR

### ISSUE THREAD

**Issue by ocdtrekkie** [prologic/todo#33]
cc: @KevinNThomas 

I am super excited about the new features, however, I wanted to report some pain points with the update:

The `-dbpath` method for setting the database location now returns an error. I am assuming/hoping an environment variable of DBPATH might work instead, given the new design of things. I haven't been able to verify that though due to my other issue.

However, my main breaking problem with the Sandstorm packaging appears to be the fact that the custom theme system doesn't work without writing a copy of the selected theme to the css folder. Sandstorm mounts the app as a read-only file system, so this is *really annoying to bypass*. I am not positive, but it doesn't look like placing the file named color-theme.css there will fix it, it looks like the app creates it or copies it no matter what options you set. This approach feels a little weird... I'm hoping there's some way Kevin knows of to set the theme that doesn't require writing a file at runtime.

**Comment by ocdtrekkie** [comment]
(Another reason it may be ideal to avoid this strategy with the CSS file, is an ideal future update would let users change the theme while the app is running, and perhaps store their preference as a setting.)

**Comment by KevinNThomas** [comment]
For the `dbpath` bug, yes setting the `DBPATH` environment variable should work, however it's weird that the flag functionality broke. I will look into that.

For the CSS themes, good point. I agree that changing the theme from within the running app would be ideal. For now though, I wonder if just moving all the color themes to the `static/css` directory and changing the logic so that it reads the correct style sheet based on the environment variable would work. I'm not totally sure how to implement that right now though, might have to do some more research.

**Comment by KevinNThomas** [comment]
https://github.com/prologic/todo/pull/34 should fix the issue you describe with the `-dbpath` flag

**Comment by ocdtrekkie** [comment]
I appreciate the prompt response! I may end up sticking with the env variables now (it doesn't matter much in my use case) though I figured as a breaking change it was an important note to fix for existing users.

I kinda hate asking for adjustments to accommodate my (sort of weird) use case, but I think the theme issue would present issues even for conventional Linux usage patterns, since prior to the update one with only r/x access to the todo directory could use -dbpath to run todo but save in their home folder. I am trying to think if I can make a temporary workaround for my Sandstorm package, but it'll probably be messy, and it's easier to maintain and keep up-to-date if we don't make any modifications to the app itself.

**Comment by ocdtrekkie** [comment]
Can the template line where color-theme.css is linked be substituted for a variable, such that it puts the correct style name there?

i.e., here, https://github.com/prologic/todo/blob/master/templates/base.html#L8 put something like https://github.com/prologic/todo/blob/master/templates/index.html#L14 ?

I am not particularly familiar with Go, (if I write a web app, it's PHP), but the general concept seems the same there.

**Comment by KevinNThomas** [comment]
Yes I think that's the right idea. I believe you have to use the `html/template` go package to set all of that up though, and I'm not familiar with that package so it might take some time to implement.

**Comment by ocdtrekkie** [comment]
Can we put the "on hold" on this?
